client/settings/app-settings.jsx

```diff
@@ -79,6 +79,11 @@ class AppForm extends React.Component {
               label='Run ShieldBattery on system startup'
               inputProps={{ tabIndex: 0 }}
             />
+            <CheckBox
```

Comment:
This should probably be disabled if the `Run ShieldBattery on system startup` setting is `false`. There's an example in some of our other settings that shows how you can reference the value of one setting in another.

Reply:
I made a whole styled Checkbox for this since it seems I can't add styles/classes directly to Checkbox. Could you point me towards the documentation for how the whole styling thing works for this project? Is it all basic electron features (or react?) or is there a theme/library (like bootstrap) that's being used?

Reply:
Yeah, that's exactly what you should've done. We use [Styled Components](https://styled-components.com/) library for styling, so you can poke around its documentation to learn a bit more about it. You can also browse through our client components to see various ways in which we have used it so far.